content/applications/finance/accounting/bank/setup/foreign_currency.rst

```diff
@@ -1,117 +1,116 @@
+   converts between the two currencies.
 
-Based on the above example, let's assume we receive the following bill
-from a supplier in China.
+Unrealized Currency Gains/Losses Report
+=======================================
 
-In the :menuselection:`Purchase --> Vendor Bills` , this is what you could see:
+This report gives an overview of all unrealized amounts in a foreign currency on your balance sheet,
+and allows you to adjust an entry or manually set an exchange rate. To access this report, go to
+:menuselection:`Reporting --> Management: Unrealized Currency Gains/Losses`. From here, you have
+access to all open entries in your **balance sheet**.
 
-.. image:: foreign_currency/foreign03.png
+.. image:: foreign_currency/foreign-gains-losses.png
    :align: center
+   :alt: View of the Unrealized Gains/Losses journal.
 
-Once you are ready to pay this bill, click on register payment on the
-bill to record a payment.
+If you wish to use a different currency rate than the one set in :menuselection:`Accounting -->
+Configuration --> Settings --> Currencies`, click the :guilabel:`Exchange Rates` button and change
+the rate of the foreign currencies in the report.
 
-.. image:: foreign_currency/foreign04.png
+.. image:: foreign_currency/foreign-exchange-rates.png
    :align: center
+   :alt: Menu to manually change exchange rates.
 
-That's all you have to do. Odoo will automatically post the foreign
-exchange gain or loss at the reconciliation of the payment with the
-invoice, depending if the currency rate increased or decreased between
-the invoice and the payment date.
+When manually changing **exchange rates**, a yellow banner appears allowing you to reset back to
+Odoo's rate. To do so, simply click on :guilabel:`Reset to Odoo's Rate`.
 
-Note that you can pay a foreign bill with another currency. In such a
-case, Odoo will automatically convert between the two currencies.
+.. image:: foreign_currency/foreign-reset-rates.png
+   :align: center
+   :alt: Banner to reset back to Odoo's rates.
 
-Customers Statements
-====================
+In order to update your **balance sheet** with the amount of the :guilabel:`adjustment` column,
+click on the :guilabel:`Adjustment Entry` button. In the pop-up window, select a
+:guilabel:`Journal`, :guilabel:`Expense Account` and :guilabel:`Income Account` to calculate and
+process the **unrealized gains and losses**.
 
-Customers and vendor statements are managed in the currency of the
-invoice. So, the amount due by your customer (to your vendor) is always
-expressed in the currency of the invoice.
+You can set the date of the report in the :guilabel:`Date` field. Odoo will automatically reverse
+the booking entry to the date set in :guilabel:`Reversal Date`.
 
-If you have several invoices with different currencies for the same
-customer, Odoo will split the customer statement by currency, as shown
-in the report below.
+Once posted, the :guilabel:`adjustment` column says 0.00€.
```

Comment:
Unless you are writing an example, I would avoid writing an amount with a specific currency. Especially since your screenshot is in dollars.

It could be something such as
... the adjustment column should indicate a `0` value, which means...